pkg/config/config.go

```diff
@@ -826,6 +828,7 @@ func load(config Config, origin string, loadSecret bool) error {
 	}
 
 	loadProxyFromEnv(config)
+	readAPIKeyFromFile(config)
```

Comment:
Here, it seems api_key_file has precedence over api_key.

Reply:
I’ve reversed the precedence to be more consistent.
Now, `api_key_file` should have precedence over `api_key` everywhere.
Setting simultaneously the two isn’t useful but having a consistent behaviour is important. There is no obvious reason for choosing one precedence order rather than the other. So, I chose the precedence order that we had in agent 5: DataDog/docker-dd-agent#208.